### PR TITLE
[Hacktoberfest] JAMES-2189 fix regexpMapping

### PR DESCRIPTION
**Pull request by strokyl**
…ppings

### CODE REVIEW

File: server/data/data-library/src/test/java/org/apache/james/rrt/lib/RecipientRewriteTableUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package org.apache.james.rrt.lib; /****************************************************************
```
Comment (chibenwa) [inline server/data/data-library/src/test/java/org/apache/james/rrt/lib/RecipientRewriteTableUtilTest.java]: Oups?

File: server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTable.java [hunk 2]
```diff
@@ -55,7 +55,6 @@ public void init() throws MessagingException {
         processor = new RecipientRewriteTableProcessor(virtualTableStore, domainList, getMailetContext());
     }
 
-
     /**
```
Comment (chibenwa) [inline server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/RecipientRewriteTable.java]: Useless diff change.

File: server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java [hunk 3]
```diff
@@ -76,6 +76,17 @@ public void tearDown() throws Exception {
     }
 
     @Test
+    public void testStoreAndGetMappings() throws
+        org.apache.james.rrt.api.RecipientRewriteTable.ErrorMappingException, RecipientRewriteTableException {
```
Comment (rouazana) [inline server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java]: why FQDN?

File: server/data/data-library/src/test/java/org/apache/james/rrt/lib/RecipientRewriteTableUtilTest.java [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import org.junit.Rule;
+import org.junit.Test;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class RecipientRewriteTableUtilTest {
+
+    @Test
+    public void regexMap() throws Exception {
```
Comment (rouazana) [inline server/data/data-library/src/test/java/org/apache/james/rrt/lib/RecipientRewriteTableUtilTest.java]: test name please (...should...when)

File: server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java (hunk 3 above)
Comment (quynhn) [inline server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java]: why full package of ErrorMappingException?

Comment (strokyl) [reply server/data/data-library/src/test/java/org/apache/james/rrt/lib/AbstractRecipientRewriteTableTest.java]: Good question I just copy and paste an already existing test, I will check why

**Comment by chibenwa** [comment]
```
[877317800e28555b6545b1759ba8d017e9735fb4] [ERROR] Errors: 
[877317800e28555b6545b1759ba8d017e9735fb4] [ERROR]   ContainerTest.containerShouldBeReachableOnExposedPort:45 » Socket Connection r...
[877317800e28555b6545b1759ba8d017e9735fb4] [INFO] 
[877317800e28555b6545b1759ba8d017e9735fb4] [ERROR] Tests run: 9, Failures: 0, Errors: 1, Skipped: 0
```

test this please

**Comment by mbaechler** [comment]
merged

**Pull request closed without merging**